### DISCUSSION TITLE
iOS version

### DISCUSSION THREAD

**Discussion by sonalder-darlene** [KyleKun/one_second_diary#65] [General]
I see that there is an iOS folder in the GitHub (haven't been updated in a while) do you still plan to release it to iPhone users or did you gave up with iOS and the AppStore ?

**Comment by KyleKun** [comment]
Hi Darlene,

The iOS folder on github was auto-generated by Flutter, and the updates on
it happened because some contributors had a Mac.

In other words, the actual implementation for the app to work on iOS still
needs to be done, and I don't have a Mac yet unfortunately, which prevents
me from doing so.

Once I am able to get a Mac I'll work on it, I just don't know how long it
will take.

Thanks for reaching out!
